bool open_terminal
---
bool access
string name
